aptos-move/framework/aptos-stdlib/sources/math_fixed.move

```diff
@@ -0,0 +1,111 @@
+/// Standard math utilities missing in the Move Language.
+module aptos_std::math_fixed {
+    use std::fixed_point32;
+    use std::fixed_point32::FixedPoint32;
+
+    const EOVERFLOW: u64 = 1;
```

Comment:
Can you add a single line comment for this? When compiling the code, error comments are used to generate messages for error codes. This leads to a human readable error message when a tx fails with that error code.